brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2966: Fix - Container text-color is taking more priority than individual heading text color of blocks.

### Description

Container text-color is not being override by the heading tags text color rest other tags color is working as intended 

### Screenshots


### Types of changes





### How has this been tested?

Steps to reproduce:

- Add a container and info box block inside 
- Set the overall content color from the container
- Set a different color for info box title
- From the backend the color changes show as they should, however the on the frontend the info box color only show the container content color but does not show the info box color set on the title

Reference: [Screen Recording 2023-06-04 at 08.22.05.40 PM](https://share.getcloudapp.com/6quJ7jyd)

### Checklist:
